# Firmware version information generated by every component

# MAJOR, MINOR, and PATCH shall be the software version number 
# and shall be defined by Sematic Versioning 2.0. 
# For build number 0000, it shall be the most recent software release.

std_msgs/Header header

uint16 mjr
uint16 mnr
uint16 patch
char build0   # ASCII character
char build1   # ASCII character
char build2   # ASCII character
char build3   # ASCII character
